packages/tao/src/shared/workspace.ts

```diff
@@ -23,40 +22,15 @@ export interface WorkspaceJsonConfiguration {
   /**
    * Projects' configurations
    */
-  projects: { [projectName: string]: ProjectConfiguration };
+  projects: {
+    [projectName: string]: ProjectConfiguration & NxJsonProjectConfiguration;
+  };
 
   /**
    * Default project. When project isn't provided, the default project
    * will be used. Convenient for small workspaces with one main application.
    */
   defaultProject?: string;
```

Comment:
I wonder if we should move this as well. It's akin to generator defaults.

Reply:
Yeah, I thought about this one a bit. It makes sense to move as it is similar to the generator defaults. I think I went with keeping it here just because it is "specific to a project" in that it points to a specific project, but at the end of the day it affects the whole workspace as far as how commands are evaluated. It should probably be moved, I'll do that and try to finish up the unit tests thursday.

Reply:
Latest push contains this